src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Input/Commands/FaultedTreeCommandGroupHandler.cs

```diff
@@ -0,0 +1,83 @@
+﻿using Microsoft.VisualStudio.Input;
+using Microsoft.VisualStudio.ProjectSystem.Input;
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Microsoft.VisualStudio.ProjectSystem.VS.Input.Commands
+{
+    internal class FaultedTreeCommandGroupHandler : IAsyncCommandGroupHandler
+    {
+        [ExportCommandGroup(CommandGroup.VisualStudioStandard97)]
+        [AppliesTo(ProjectCapability.CSharpOrVisualBasic)]
+        [Order(int.MaxValue)] // We want to be ahead of the standard handlers
+        public static IAsyncCommandGroupHandler VisualStudioStandard97Handler =>
```

Comment:
I don't think so: there's no way to tell what command group the command is coming from in the `IAsyncCommandGroupHandler` methods, and unless I'm mistaken ID's aren't unique across groups.